game/scripts/npc/abilities/obsidian_destroyer_equilibrium.txt

```diff
@@ -41,7 +41,7 @@
       "04"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "duration"                                        "7"
+        "duration"                                        "7 8 9 10 11 12" //OAA
```

Comment:
mana steal needs a buff early game but doesn't need to scale as much. Leave duration at 7.
Mana steal should be 90/110/130/150/160/170 or something like a constant 125 at all levels would be fine.